addons/survey/views/survey_templates.xml

```diff
@@ -484,34 +494,36 @@
                         <t t-if="survey.description"><div t-field='survey.description' class="oe_no_empty"/></t>
                         <t t-call="survey.retake_survey_button"/>
                     </div>
-                    <div role="form" class="js_surveyform" t-att-name="'%s' % (survey.id)" t-att-data-prefill="'/survey/prefill/%s/%s' % (survey.access_token, answer.token)">
-                        <t t-foreach='survey.question_and_page_ids' t-as='question'>
-                            <hr t-if="question.is_page and question != survey.page_ids[0]" />
-                            <div t-if="question.is_page" class="o_page_header">
-                                <h1 t-field='question.title' />
-                                <div t-if="question.description" t-field='question.description' class="oe_no_empty"/>
-                            </div>
-                            <t t-if="not question.is_page and not answer or (question in answer.question_ids)" >
-                                <t t-set="prefix" t-value="'%s_%s' % (survey.id, question.id)" />
-                                <div class="js_question-wrapper" t-att-id="prefix">
-                                    <h2>
-                                        <span t-field='question.question' />
-                                        <span t-if="question.constr_mandatory" class="text-danger">*</span>
-                                        <span t-if="quizz_correction" class="badge badge-pill" t-att-data-score-question="question.id"></span>
-                                    </h2>
-                                    <t t-if="question.description"><div class="text-muted oe_no_empty" t-field='question.description'/></t>
-                                    <t t-if="question.question_type == 'free_text'"><t t-call="survey.free_text"/></t>
-                                    <t t-if="question.question_type == 'textbox'"><t t-call="survey.textbox"/></t>
-                                    <t t-if="question.question_type == 'numerical_box'"><t t-call="survey.numerical_box"/></t>
-                                    <t t-if="question.question_type == 'date'"><t t-call="survey.date"/></t>
-                                    <t t-if="question.question_type == 'datetime'"><t t-call="survey.datetime"/></t>
-                                    <t t-if="question.question_type == 'simple_choice'"><t t-call="survey.simple_choice"/></t>
-                                    <t t-if="question.question_type == 'multiple_choice'"><t t-call="survey.multiple_choice"/></t>
-                                    <t t-if="question.question_type == 'matrix'"><t t-call="survey.matrix"/></t>
-                                    <div class="js_errzone alert alert-danger" style="display:none;" role="alert"></div>
+                    <div role="form" class="js_surveyform o_survey_form">
```

Comment:
Do you really need `js_surveyform` AND `o_survey_form` ?
Maybe remove js_surveyform by replacing the necessary references in the css.